src/v2/Apps/Order/redirects.tsx

```diff
@@ -91,6 +91,11 @@ const goToStatusIfNotOfferOrder = goToStatusIf(
   "Not an offer order"
 )
 
+const goToStatusIfNotCreditCardOffer = goToStatusIf(
```

Comment:
**suggestion (non-blocking):** `goToStatusIfBuyNowCreditCardOrder` would be a more accurate name for this rule. (We should also clean up the old `goToStatusIfNotOfferOrder` rule.